cmd/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	_ "github.com/ModelarData/Telegraf-Output-Apache-Arrow-Flight/plugins/output/flight"
+
+	"github.com/influxdata/telegraf/plugins/common/shim"
+)
+
+func main() {
+
+	var pollInterval = flag.Duration("poll_interval", 1*time.Second, "how often to send metrics(in seconds)")
+	var pollIntervalDisabled = flag.Bool("poll_interval_disabled", false, "set to true to disable polling. You want to use this when you are sending metrics on your own schedule")
```

Comment:
"You want to use this when" -> "Use this when" ?

Reply:
Fixed.